src/jobs/jobs.service.ts

```
@@ -191,6 +195,16 @@ export class JobsService {
     return application;
   }
 
+  async createApplication(dto: CreateApplicationDto): Promise<Application> {
+    const jobId = typeof dto.jobId === 'string' ? parseInt(dto.jobId, 10) : dto.jobId;
+    const job = await this.jobRepository.findOne({ where: { id: jobId } });
+    if (!job) throw new NotFoundException(`Job with ID ${dto.jobId} not found`);
+    if (!job.isAcceptingApplications)
+      throw new ForbiddenException('This job is not accepting applications.');
+    const app = this.applicationRepository.create(dto);
+    return this.applicationRepository.save(app);
+  }
```

Biome (1.9.4)

[error] Duplicate class member name "createApplication"



(lint/suspicious/noDuplicateClassMembers)